index.html

```diff
@@ -3515,10 +3515,10 @@ <h5>JWT Encoding</h5>
             </p>
 
             <p>
-If a JWS is present, the digital signature either refers to the <a>issuer</a>
+If a JWS is present, the digital signature refers either to the <a>issuer</a>
 of the <a>verifiable credential</a>, or in the case of a
-<a>verifiable presentation</a>, the <a>holder</a> of the
-<a>verifiable credential</a>. The JWS proves that the <a>issuer</a> of the JWT
+<a>verifiable presentation</a>, to the <a>holder</a> of the
+<a>verifiable credential</a>. The JWS proves that the <code>iss</code> of the JWT
 signed the contained JWT payload and therefore, the <code>proof</code>
 <a>property</a> can be omitted.
```

Comment:
since I have never seen proof included, we might consider making `can` a SHOULD or MUST at this point.

Reply:
can was used so that the change could go into 1.1 without requiring any changes to the mandatory requirements. If this restriction has now been dropped on v1.1 then we can indeed start to insert MUST, MUST NOT etc as appropriate

Reply:
if normative language is added to this PR it would need to be labelled `defer-v2`